setup.cfg

```
@@ -120,7 +121,8 @@ compression =
 # `all ` should be kept as the complete set of pandas optional dependencies for general use.
 all =
     beautifulsoup4>=4.9.3
-    blosc>=1.21.0
+    # blosc only available on conda
```

To save future readers' time it would be helpful to point to the evidence/discussion on this point
```suggestion
    # blosc only available on conda (https://github.com/Blosc/python-blosc/issues/297)
```